magento-research/pwa-studio
# bug: Unwanted content displayed in search suggestion box when there are no results for search

## Description
Disable render suggestion component when search query have not results.

## Related Issue




Closes #1223.

## Verification Steps

1. Go to Venia store front and click on search icon.
2. Enter search string which should not return any results. example - afgdsgedf
3. Search suggestion box display "No results were found."

## How Have YOU Tested this?
Verification steps above!

## Screenshots / Screen Captures (if appropriate):
![Screenshot from 2019-05-17 15-48-56](https://user-images.githubusercontent.com/1911347/57915906-fbc94580-78bb-11e9-84ba-2c365d48da97.png)
